src/components/buttons/_private/ButtonBase/ButtonBase.scss

```diff
@@ -79,6 +79,10 @@ $disabled-text-dark: $gray-dark50;
 		}
 	}
 
+	&:focus {
+		@include defaultOutline;
+	}
+
```

Comment:
I think it might be worth calling out both in this PR (and the product review testers later) that the focus styles are specifically global and affect every instance of these components even outside the scope of this work (the add site form).